### ISSUE TITLE
[Product] Create Sub-Domains for .OneMillionDevs.ETH

### ISSUE THREAD

**Issue by kamescg** [OneMillionDevs/ideas#1]
# User Story
As a One Million Developer Steward I want to more easily onboard developers into Ethereum and the 1 Million Developers initiative using the Ethereum Name System (https://ens.domains/) and the W3C Decentralized Identities specification (https://www.w3.org/TR/did-core/)

# Abstract
A sub-domain and decentralized identity document are powerful.

They create the foundation for developer profiles, peer-to-peer verifiable credentials, enhanced security through easy-to-read wallet addresses, and many other benefits.

A potential product for the One Million Developers could be a tool to both register an Ethereum sub-domain and also create a decentralized developer identity/profile.

# Technical Requirements

- Register `onemilliondevs.eth` sub-domain
- Stake OMD tokens to get domain?
- Pay to get sub-domain? Funds public utility?
- Add decentralized document to ENS TXT record? 

https://github.com/ensdomains/subdomain-registrar/blob/master/README.md

# Notes

## Coogan from ConsenSys
ELI5 TLDR: ENS domain names (such as onemilliondevs.eth) can be stand-ins for hexadecimal addresses (the typical way ethereum accounts are represented), you can read more here [0].

If you own an ENS domain, it’s relatively easy to create subdomains (like coogan.onemilliondevs.eth), which can also stand-in for ethereum accounts. this can be a similar identifier to an institutional email (you know i work at Consensys because i have an email that ends in @consensys.net) and a way to track people using blockchain.

# References
https://medium.com/@eric.conner/the-ultimate-guide-to-ens-names-aa541586067a
https://www.youtube.com/watch?v=qF2lhJzngto
https://medium.com/universal-ethereum/universal-logins-first-demo-1dc8b17a8de7
https://link.medium.com/j1HKZ9o6f1
https://twitter.com/pet3rpan_/status/1199769132637442048?s=20

# Estimate
**Time:** 2 Weeks to 1 Month
**Cost:** $3,000

**Comment by orbro** [comment]
@KamesCG Did one of us buy onemilliondevs.eth? I have milliondevs.eth if we need a backup.

**Comment by kamescg** [comment]
@orbro No. I believe it was purchased around the time of the announcement at Devcon.
